tst/EnergyPlus/unit/BoilerHotWater.unit.cc

```diff
@@ -240,12 +240,10 @@ TEST_F(EnergyPlusFixture, Boiler_HotWater_BoilerEfficiency)
 
     state->dataPlnt->PlantLoop.allocate(state->dataPlnt->TotNumLoops);
     for (int l = 1; l <= state->dataPlnt->TotNumLoops; ++l) {
-        auto &loop(state->dataPlnt->PlantLoop(l));
-        loop.LoopSide.allocate(2);
```

Comment:
All the `.allocate() `calls went away because we converted LoopSide to `std::array` which is fixed size.